Feature/intygfv 11523

perbeijer opened the pull request.

perbeijer committed:
INTYGFV-11523: Slå ihop tabellerna för invanare och vardenhet med bestallning

perbeijer committed:
INTYGFV-11523: Uppdaterat felmeddelanden för visa beställning

an3ll commented on web/src/main/java/se/inera/intyg/intygsbestallning/web/controller/testability/BestallningResource.java:
```diff
@@ -31,52 +23,24 @@
     public Response deleteBestallning(@PathVariable("id") Long id) {
         return entityTxMapper.jsonResponse(() -> {
-            var bestallning = bestallningPersistenceService.getBestallningByIdAndHsaIdAndOrgId(id, null, null).orElseThrow(
+            var bestallning = bestallningPersistenceService.getBestallningById(id).orElseThrow(
```
Använd BestallningRepository här direkt istället för att skapa en ny metod som enbart finns för testsyfte.

perbeijer committed:
INTYGFV-11523: Testability-api för beställningar går nu direkt mot BestallningRepository

perbeijer committed:
INTYGFV-11523: Merge branch 'develop' into feature/INTYGFV-11523

# Conflicts:
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/repository/InvanareRepository.java
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/repository/VardenhetRepository.java
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/service/InvanarePersistenceService.java
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/service/InvanarePersistenceServiceImpl.java
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/service/VardenhetPersistenceService.java
#	persistence/src/main/java/se/inera/intyg/intygsbestallning/persistence/service/VardenhetPersistenceServiceImpl.java
#	web/src/main/java/se/inera/intyg/intygsbestallning/web/controller/testability/BestallningResource.java
#	web/src/main/java/se/inera/intyg/intygsbestallning/web/service/bestallning/CreateBestallningServiceImpl.java
#	web/src/main/java/se/inera/intyg/intygsbestallning/web/service/bestallning/VisaBestallningServiceImpl.java

perbeijer committed:
INTYGFV-11523: Codequality
